src/openms/source/FORMAT/FASTAFile.cpp

```diff
@@ -198,9 +198,9 @@ namespace OpenMS
     }
     ++entries_read_;
 
-    protein.identifier = std::move(id_);
-    protein.description = std::move(description_);
-    protein.sequence = std::move(seq_);
+    protein.identifier = id_;
+    protein.description = description_;
```

Comment:
maybe worth to check why clang-tidy did remove the std::move here (aka has no effect)

Reply:
@timosachsenberg I would not follow through with these, first it removes intent and secondly copying or moving-ing `std::string` clearly has a different effect! removing `std::move` may have a performance penalty here

Reply:
Sure. But according to clang it only removes the std::move if it has no effect. So it is worth to investigate why it gets removed while it shouldn’t

Reply:
not sure here. it is a normal std::string stored in a member.